mail/views.py

```diff
@@ -52,3 +54,34 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-
                 } for batch_num, resp in enumerate(mailgun_responses)
             }
         )
+
+
+class FinancialAidMailView(APIView):
+    """
+    View for sending financial aid emails to individual learners
+    """
+    authentication_classes = (authentication.SessionAuthentication, )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, UserCanEditFinancialAid)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Post request to send emails to an individual learner
+        """
+        try:
+            subject = request.data.get('email_subject') or kwargs['email_subject']
```

Comment:
Please only use `request.data` to get information here. We shouldn't use this function except to respond to requests. If there is substantial functionality which can be shared between other functions we should make a separate function to share that functionality.